VKApp/VKApp/Views/PhotosUserCollectionViewCell.swift

```diff
@@ -14,4 +23,40 @@ final class PhotosUserCollectionViewCell: UICollectionViewCell {
     func configureCell(userPhoto: String) {
         friendPhotoImageView.image = UIImage(named: userPhoto)
     }
+
+    func animateShowFriendPhotoImageView() {
+        let animationScaleX = CABasicAnimation(keyPath: Constants.transformScaleXText)
+        animationScaleX.fromValue = 0
+        animationScaleX.toValue = 1
+        animationScaleX.duration = 1
```

Comment:
```suggestion
func animateScaleX(fromValue, toValue, duration)
```
чтобы не писать каждый раз по 3 строчки, и вызывать в каждой вью